dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -446,28 +446,42 @@ def test_null_count(self):
 
 class TestProfilerNullValues(unittest.TestCase):
 
-    @classmethod
-    def setUpClass(cls):
```

Comment:
Got rid of this/reorganized it because it doesnt really make sense to have a setupClass function for one test.

Reply:
Since we are updating this, I think the profiler should not run the datalabeler, will make it faster.